tests/instances.tcl

```diff
@@ -287,6 +294,7 @@ proc parse_options {} {
             puts "--fail                  Simulate a test failure."
             puts "--valgrind              Run with valgrind."
             puts "--tls                   Run tests in TLS mode."
+            puts "--tls-mod               Run tests in TLS mode with Redis module."
```

Comment:
I think this is a bit confusing, why not use `--tls` to enable TLS tests and `--tls-mod` (or `--tls-module` so it's more obvious) to both enable TLS and load the module?